client/mass/test/regression.spec.js

```diff
@@ -5,51 +5,45 @@ const raceGroupsetting = {
 	id: 'genetic_race',
```

Comment:
todo later: should deepFreeze() these tw objects that are reused in tests, to ensure that no tests are rewriting/mutating props by accident or intentionally